adoorback/account/views.py

```diff
@@ -24,7 +24,7 @@
 from safedelete.models import SOFT_DELETE_CASCADE
 
 from .email import email_manager
-from .models import Subscription
+from category.models import Category, Subscription
```

Comment:
I don't think this is handled yet,
I meant you should get rid of SubscribeUserContent and UnsubscribeUserContent since it's related to the deprecated subscription model.
Let me know if I misunderstood anything!

Reply:
Got it, that was my misunderstanding from before -- I've removed everything related to those two from `account/views.py` and `account/urls.py`